src/formatters/needlessDisablesStringFormatter.js

```diff
@@ -0,0 +1,25 @@
+import chalk from "chalk"
+import path from "path"
+
+function logFrom(fromValue) {
+  if (fromValue.charAt(0) === "<") return fromValue
+  return path.relative(process.cwd(), fromValue).split(path.sep).join("/")
+}
+
+export default function (report) {
+  let output = ""
+
+  report.forEach(sourceReport => {
```

Comment:
At the moment, this outputs the name of each source checked irregardless of whether needless disables were found within it. Was this in intentional? 

In contrast, the warnings string reporter returns an empty string if no infractions were found. Do we want a consistent behaviour here?